lib/cryptoexchange/helpers/hash_helper.rb

```diff
@@ -0,0 +1,20 @@
+module HashHelper
+  class << self
+    # "Polyfill" of the Hash#dig method from Ruby 2.3
+    #
+    #   Examples
+    #
+    #     > dig({a: {b: 1}}, :a, :b)
+    #     => 2
+    #
+    #     > dig({a: {b: 1}}, :a, :c)
+    #     => nil
+    def dig(obj, *args)
```

Comment:
What is the lowest version of Ruby where `dig` is supported? `2.3` yes?
I would prefer to keep `2.2` for now if possible, same argument which is to support as wide as possible until it is a problem that we need to bump to 2.3
However, forcing 2.3 earlier would also mean we do not need to backward compatible a lot of things later on when they accumulate.